research/sentiment_analysis/sentiment_main.py

```diff
@@ -0,0 +1,188 @@
+"""The model makes use of concatenation of two CNN layers with different kernel sizes.
+`sentiment_model.py` for more details about the models.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# pylint: disable=g-bad-import-order
+from absl import app as absl_app
+from absl import flags
+import tensorflow as tf
+# pylint: enable=g-bad-import-order
+
+from data import dataset
+from data.dataset import imdb
+import sentiment_model
+from official.utils.flags import core as flags_core
+from official.utils.logs import hooks_helper
+from official.utils.logs import logger
+
+def evaluate_model(estimator, num_gpus):
```

Comment:
nit: this evaluate function is simple enough to be defined in the main training cycle, just like the train.